composer.json

```diff
@@ -7,9 +7,9 @@
     "prefer-stable": true,
     "require": {
         "brianium/paratest": "*",
-        "friendsofphp/php-cs-fixer": "^2.18.2",
+        "friendsofphp/php-cs-fixer": "^3.5",
         "nikic/php-parser": "^4.3",
-        "phpro/grumphp": "^1.4",
+        "phpro/grumphp": "^1.5",
```

Comment:
why dont we go for shim instead? Been using it in quite some projects without any issues, and it is easier to maintain

Reply:
Was actually not part of the PR, but I would prefer the shim as well.

Reply:
Feel free to make a separate PR for that.

Reply:
fyi: updated this to the shim version already in this PR: https://github.com/janvernieuwe/conventions/blob/b02a513f55344f079b7b4b7270b79c75cb29d489/composer.json#L12